site/viamguides/src/workshop-plant-watering/workshop-plant-watering.md

```diff
@@ -311,93 +310,49 @@ Duration: 40
-   ```bash
-   $ which python3
-   ```
-1. Run the following command from your computer (not the SSH prompt to your Raspberry Pi) to copy the code from your computer to your Raspberry Pi. In the command, you will copy `process.py` over to your Raspberry Pi, with the section following the colon `:` indicating where your file should be copied to on the Raspberry Pi (the path of the directory you are working in on your Raspberry Pi, along with the filename).
-   ```bash
-   $ scp process.py user@host.local:/home/myboard/process/process.py
-   ```
+You can find the source code and documentation for the service on GitHub: [https://github.com/viam-devrel/watering-controller](https://github.com/viam-devrel/watering-controller)
 
 <form>
-  <name>What is the primary role of the `process.py` script in the plant watering system?</name>
+  <name>What is the primary role of the `watering-controller` service in the plant watering system?</name>
   <input type="radio" value="To display moisture sensor readings directly on the Raspberry Pi.">
   <input type="radio" value="To define the logic for periodically checking sensor readings and toggling the pump.">
   <input type="radio" value="To update the firmware of the moisture sensor and motor pump.">
   <input type="radio" value="To configure the network settings for the Raspberry Pi.">
 </form>
 
-### Configure a Viam process
-
-1.  Now let's allow `viam-server` to run the process as the root user on your Raspberry Pi by configuring a [Viam process](https://docs.viam.com/configure/processes/). In [the Viam app](https://app.viam.com/fleet/locations) under the **CONFIGURE** tab, click the **+** icon in the left-hand menu and select **Process**.
-1.  Find the corresponding card to `process-1`. Enter the executable path of Python3 running on your Raspberry Pi that you output from a previous step. Add an argument of the `process.py` file to run on your Raspberry Pi. Enter the working directory where you want the process to execute.
-    ![configure process](assets/configureProcess.png)
-1.  Still within the `process-1` card, select the advanced settings icon near the top right corner to review the configuration JSON. Create a new `env` property, and add your environment variables within the new property, formatted like the following with your own credentials.
-    ```json
-      "env": {
-        "BOARD_NAME": "board-1",
-        "ROBOT_API_KEY": "your-api-key",
-        "ROBOT_API_KEY_ID": "your-api-key-id",
-        "ROBOT_ADDRESS": "your-robot-address"
-      },
-    ```
-    ![configure JSON](assets/configureJSON.png)
-    > aside negative
-    > The `BOARD_NAME` is the default name for the Raspberry Pi when added to our Viam machine. Other machine credentials can be found under the **CONNECT** tab, selecting an SDK, and toggling **Include API key** to reveal your credentials within the code sample.
-        ![get credentials](assets/getCredentials.png)
```

Comment:
Since we are deleting this section, should we also remove this image (getCredentials.png) from the workshop assets?